attributes/default.rb

```diff
@@ -34,7 +36,7 @@
 default["sensu"]["apt_repo_url"] = "http://repositories.sensuapp.org/apt"
 default["sensu"]["yum_repo_url"] = "http://repositories.sensuapp.org"
 default['sensu']['yum_flush_cache'] = nil
-default["sensu"]["msi_repo_url"] = "https://repositories.sensuapp.org/msi"
+default["sensu"]["msi_repo_url"] = "https://repositories.sensuapp.org/msi/2012r2"
```

Comment:
why does this need to change? I am still unclear with the previous discussion wont this break all other versions?

Reply:
@majormoses When you go to the old url, version go up to 0.26. Logrotate works as of 0.29.x. Puppet version also uses the new url, however still using the 0.29.x version. If uncertain of my comment please check the old url vs new url.

Reply:
I see new versions in here: http://repositories.sensuapp.org/msi/2016/ which was my point the windows version plays into what is installed. I really don't know enough about windows stuff but this feel like this would break every version other than server 2012.

Reply:
I think the problem is that we need to include the `version` not hardcore it to `2012r2`

I don't have a windows machine to test with can you please give me the relevant ohai output for this: https://github.com/chef/ohai/blob/v14.1.0/lib/ohai/plugins/windows/platform.rb#L30 as I think we can use this to solve the problem.